casepro/cases/models.py

```diff
@@ -435,6 +490,7 @@ def as_json(self, full=True):
             return {
                 'id': self.pk,
                 'assignee': self.assignee.as_json(full=False),
+                'user_assignee': self.user_assignee.as_json(full=False) if self.user_assignee else None,
```

Comment:
Resolved with https://github.com/rapidpro/casepro/pull/130/commits/c6b66b7e9c5351201946163a08aec0fd63765b30 and https://github.com/rapidpro/casepro/pull/130/commits/e50e46195efdad3bd522143d82d37d3912167a55